packages/web3torrent/src/components/torrent-info/channels-list/ChannelsList.tsx

```diff
@@ -41,17 +41,24 @@ class ChannelsList extends React.Component<UploadInfoProps> {
         wire.paidStreamingExtension.pseChannelId === channelId
     );
 
-    const uploaded = wire ? wire.uploaded : 0;
-    const peerAccount =
-      pseType === 'leecher' ? channels[channelId].beneficiary : channels[channelId].payer;
+    let transferred: string;
+    let peerAccount: string;
+    if (wire) {
+      transferred = pseType === 'seeder' ? prettier(wire.uploaded) : prettier(wire.downloaded);
+      peerAccount =
+        pseType === 'leecher' ? channels[channelId].beneficiary : channels[channelId].payer;
+    } else {
+      transferred = 'NOWIRE';
+      peerAccount = 'NOWIRE';
```

Comment:
We tend to lose the `wire` when there haven't been any requests recently. This UI should probably be seen as a stop-gap until we have a robust way of dealing with that.

Suggestions: 
i. Maintain some "memory" in this component: do not overwrite the current fields if the props change to `undefined`